.github/workflows/run-ci-cd.yaml

```
@@ -588,69 +537,19 @@ jobs:
-
-      - name: Upload SBOMs
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          gh release upload "${{ github.event.release.tag_name }}" \
-            backend-sbom-${{ env.RELEASE_VERSION }}.cdx.json \
-            frontend-sbom-${{ env.RELEASE_VERSION }}.cdx.json
-    timeout-minutes: 10
+      actions: write
+      contents: read
+    uses: ./.github/workflows/scan-images.yaml
```

actionlint (1.7.12)

[error] input "aws_role_session_name" is required by "./.github/workflows/scan-images.yaml" reusable workflow

(workflow-call)